honeycombio/data_source_trigger_recipient.go

```diff
@@ -28,6 +28,7 @@ func dataSourceHoneycombioSlackRecipient() *schema.Resource {
 				Optional: true,
 			},
 		},
+		DeprecationMessage: "Use honeycombio_recipient data_source instead",
```

Comment:
Love this! I think it is worth adding to the trigger_recipient data source as a doc note as well